includes/backdrop_upgrade_status.views.inc

```diff
@@ -24,38 +24,65 @@ function views_backdrop_upgrade_status_report($todo_status = FALSE) {
   $collapsed = TRUE;
   $views = views_get_all_views();
   foreach ($views as $view) {
-    switch ($view->type) {
-      case t('Normal'):
-      case t('Overridden'):
+    // Mark disabled views with a warning
+    if ( $view->disabled ) {
+      $type = t('Disabled');
+      $row = array(
+        'data' => array(
+          'name' => $view->human_name,
+          'machine' => $view->name,
+          'state' => $type,
+          'description' => $view->description,
+          'recommendation' => '',
+        ),
+      );
+      if ($type == t('Disabled')) {
```

Comment:
I don't like to do comparisons against the result of `t()`, so I'd create a `$disabled = TRUE` variable instead. I am going to merge your PR though, and make that change after :)